auto3dseg/algorithm_templates/swinunetr/scripts/algo.py

```diff
@@ -14,77 +14,120 @@
+                'transforms_infer.yaml': transforms_infer
                 }
-            )
-        patch_size = [96, 96, 96]
-        max_shape = data_cfg["stats_summary#image_stats#shape#max"]
-        patch_size = [
-            max(64, shape_k // 64 * 64) if shape_k < p_k else p_k for p_k, shape_k in zip(patch_size, max_shape)
-        ]
-        self.cfg["patch_size"] = [patch_size[0], patch_size[1], patch_size[2]]
-        self.cfg["patch_size_valid"] = [patch_size[0], patch_size[1], patch_size[2]]
-
-        modality = data_src_cfg.get("modality", "ct").lower()
-        spacing = data_cfg["stats_summary#image_stats#spacing#median"]
-
-        intensity_upper_bound = float(data_cfg["stats_summary#image_foreground_stats#intensity#percentile_99_5"])
-        intensity_lower_bound = float(data_cfg["stats_summary#image_foreground_stats#intensity#percentile_00_5"])
-        ct_intensity_xform = {
-            "_target_": "Compose",
-            "transforms": [
-                {
-                    "_target_": "ScaleIntensityRanged",
-                    "keys": "@image_key",
-                    "a_min": intensity_lower_bound,
-                    "a_max": intensity_upper_bound,
-                    "b_min": 0.0,
-                    "b_max": 1.0,
-                    "clip": True,
-                },
-                {"_target_": "CropForegroundd", "keys": ["@image_key", "@label_key"], "source_key": "@image_key"},
-            ],
-        }
-        mr_intensity_transform = {
-            "_target_": "NormalizeIntensityd",
-            "keys": "@image_key",
-            "nonzero": True,
-            "channel_wise": True,
-        }
-        for key in ["transforms_infer", "transforms_train", "transforms_validate"]:
-            for idx, xform in enumerate(self.cfg[f"{key}#transforms"]):
-                if isinstance(xform, dict) and xform.get("_target_", "").startswith("Spacing"):
-                    xform["pixdim"] = deepcopy(spacing)
-                elif isinstance(xform, str) and xform.startswith("PLACEHOLDER_INTENSITY_NORMALIZATION"):
-                    if modality.startswith("ct"):
-                        self.cfg[f"{key}#transforms#{idx}"] = deepcopy(ct_intensity_xform)
-                    else:
-                        self.cfg[f"{key}#transforms#{idx}"] = deepcopy(mr_intensity_transform)
-
-        override_params = _update_args(**override)
-        for k, v in override_params.items():
-            self.cfg[k] = v
 
+        else:
+            fill_records = self.fill_records
+
+        for yaml_file, yaml_contents in fill_records.items():
+            file_path = os.path.join(output_path, 'configs', yaml_file)
+
+            parser = ConfigParser(globals=False)
+            parser.read_config(file_path)
+            for k, v in yaml_contents.items():
+                if k in kwargs:
+                    parser[k] = kwargs[k]
+                else:
+                    parser[k] = deepcopy(v)  # some values are dicts
```

Comment:
After this line, we may need one more deepcopy to 
1) make sure fill_records do not have any memory address.
2) reflect the changes made by override paeans

```python

yaml_contents[k] = deepcopy(parser[k])

```